libs/libcds/include/hpx/libcds/hpx_tls_manager.hpp

```diff
@@ -0,0 +1,119 @@
+// Copyright (c) 2020 Weile Wei
+//
+// Distributed under the Boost Software License, Version 1.0. (See accompanying
+// file LICENSE or copy at http://www.boost.org/LICENSE_1_0.txt)
+
+#ifndef LIBCDS_HPX_TLS_MANAGER
+#define LIBCDS_HPX_TLS_MANAGER
+
+#include <hpx/config.hpp>
+#include <hpx/errors/exception.hpp>
+//
+#include <cds/gc/details/hp_common.h>
+#include <cds/gc/hp.h>
+#include <cds/threading/details/cxx11_manager.h>
+
+#include <atomic>
+
+//@cond
```

Comment:
Not a big thing: we use the `\cond` doxygen keyword syntax everywhere else (instead of `@cond`). I appreciate it if you wouldn't mind changing this.